common/lib/xmodule/xmodule/js/src/video/09_play_pause_control.js

```diff
@@ -28,11 +28,9 @@ define('video/09_play_pause_control.js', [], function() {
             '<button class="control video_control play" aria-disabled="false" title="',
                 gettext('Play'),
             '">',
-                '<span class="icon-fallback-img">',
-                    '<span class="icon fa fa-play" aria-hidden="true"></span>',
-                    '<span class="sr control-text">',
-                        gettext('Play'),
-                    '</span>',
+                '<span class="icon fa fa-play" aria-hidden="true"></span>',
+                '<span class="sr control-text">',
+                    gettext('Play'),
```

Comment:
Since this button has a `title` we can simplify this and remove the `sr` text within it.